CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
-# 1.4.0
+# 1.4.2
```

Comment:
You'll want to add the new version above this one with notes about any changes included in that release. So it would look something like this:

```markdown
# 1.4.2

- Support for getOrders and getOrderItems [#15](https://github.com/patterninc/muffin_man/pull/15)

# 1.4.0

- Pass cache key to client for caching access tokens [#11](https://github.com/patterninc/muffin_man/pull/11)

```